nox/sessions.py

```diff
@@ -795,6 +793,20 @@ def _create_venv(self) -> None:
 
         self.venv.create()
 
+    def reuse_existing_venv(self) -> bool:
+        return any(
+            (
+                # forces every session to reuse its env
+                self.global_config.reuse_venv == "always",
+                # sessions marked True will always be reused unless never is specified
+                self.func.reuse_venv is True
+                and self.global_config.reuse_venv != "never",
+                # session marked False will never be reused unless always is specified
```

Comment:
This comment doesn't relate to the code below

Reply:
I tried to clarify this function with even more documentation here 715bc9a63d2098968c176bb154407a06ddb890c3, let me know if it's better.

Reply:
This is super helpful